newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -181,21 +177,20 @@ const getOptionLabel = (option: Option, value: string): string =>
 export default React.forwardRef<Props, SemiControlledAutoCompleteInterface>(
   function SemiControlledAutoComplete(props: Props, ref) {
     const input = React.useRef((null: ?HTMLInputElement));
-    const [state, setState] = useState<State>({ inputValue: null });
-    const [isMenuOpen, setIsMenuOpen] = useState<boolean>(true);
+    const [inputValue, setInputValue] = useState((null: string | null));
```

Comment:
`null: ?string` gives a lot of flow errors due to type void, so I used this instead. Also, I tried resolving those errors, but then we could never have an empty input value in the text field (let's say for "Players" if you try to delete it, you'd get the last "P", and deleting this "P" will make the text "Players" again).

Reply:
Strange because `?string` is equivalent to `string | null | undefined`.

Anyway, not a big deal because `string | null` is actually a bit better.